modules/nw-networkpolicy-allow-internet.adoc

```diff
@@ -0,0 +1,107 @@
+// Module included in the following assemblies:
+//
+// * networking/network_security/network_policy/nw-networkpolicy-full-multitenant-isolation.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-networkpolicy-allow-ingress_{context}"]
+= Creating an allow ingress access network policy
+
+With the `deny-by-default` network policy in place, no pods can talk to each other or receive traffic from external sources. One option to enable communication is to allow some pods to receive traffic. To do so, you can create the following `ingress-access` network policy. With this network policy, pods with the  `networking/allow-ingress-access=true` label can receive network traffic.
```

Comment:
```suggestion
With the `deny-by-default` network policy, that denies ingress and egress, in place, no pods can talk to each other or receive traffic from external sources. One option to enable communication is to allow some pods to receive traffic. To do so, you can create the following `ingress-access` network policy. With this network policy, pods with the  `networking/allow-ingress-access=true` label can receive network traffic.
```

Reply:
@stevsmit If it is both ingress or egress pod traffic, it has to be default deny ingress and egress.
If it is about **ingress** then it should just refer to incoming traffic.

Reply:
Fixed.